src/main/resources/generator/dependencies/common/package.json

```
@@ -35,6 +35,7 @@
     "mkdirp": "3.0.1",
     "node": "20.17.0",
     "npm": "10.8.3",
+    "npm-run-all": "4.1.5",
```

https://www.npmjs.com/package/npm-run-all2
```suggestion
    "npm-run-all2": "6.2.2",
```